### ISSUE TITLE
No package for Ubuntu 19.04 (Disco)

### ISSUE THREAD

**Issue by vidarw** [Azure/azure-cli#9101]
**Is your feature request related to a problem? Please describe.**
Ubuntu is officially releasing version 19.04 (Disco Dingo) within a week.
There is currently no package available for this version in the Microsoft repository.
The old 18.10 package is still installed from before upgrading from 18.10 is still present and works. But all matters related to the Microsoft repository fails in 19.04 (install, update, upgrade, etc.).

**Describe the solution you'd like**
The installation instructions of Azure CLI on Debian/Ubuntu should work on Ubuntu 19.04.

**Comment by marstr** [comment]
Thanks for the reminder on this, @vidarw.

**Comment by dalareo** [comment]
The only solution I found it to aggregate Bionic repo instead...

**Comment by ctktfg** [comment]
I have used the Cosmic repo and that works as well.  Would love a real fix though.  Pretty please?  :)

**Comment by marstr** [comment]
We've now published 2.0.63 to a repository made for Disco. Can @vidarw, @dalareo, or @ctktfg confirm that it's available now?

**Comment by ctktfg** [comment]
Worked for me in the sense that it installs.  However I now get this error:
```
/opt/az/lib/python3.6/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.2) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
```

I am able to perform the first couple of basic queries that I could think of without issue.  Should I put the error above in a separate issue?

**Comment by marstr** [comment]
Yes please! It doesn't look like the fix will be available for 2.0.64 which is getting cut from the dev branch later today, but I should be able to get that fixed by 2.0.65.

**Comment by marstr** [comment]
Closing this out as the original issues is fixed.
